### PR TITLE
Feature/paypal ppcp

### PR DESCRIPTION
**Pull request by TariqAleem**
**PayPal Commerce Platform - Integrated Features**

- PayPal Checkout: Capture only

- PayPal Checkout: Authorize and Capture

- Advanced Card Payments

- Checkout using Billing Agreements

### CODE REVIEW

**Comment by jessiagee** [comment]
@TariqAleem Looks like Travis CI build failed with checks

**Review by jessiagee — CHANGES_REQUESTED** [review]
Add .idea to your global .gitignore file so that they aren't picked up by git to add to the project - https://stackoverflow.com/questions/39013880/how-to-set-global-gitignore-file-in-android-studio/41705045#41705045

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@ Gemfile*.lock
 .rvmrc
 /doc/
 .ruby-version
+.idea/*
```
[the remark this answers is not in the document]
Comment (TariqAleem) [reply .gitignore]: Hello @jessiagee , we have currently removed .idea/ from .gitignore, so should we add it ?

File: .ruby-gemset [hunk 2]
```diff
@@ -0,0 +1 @@
+active_merchant_ppcp
```
[the remark this answers is not in the document]
Comment (TariqAleem) [reply .ruby-gemset]: Yes this can be removed.

**Review by TariqAleem — COMMENTED** [review]
Changes done please review

**Comment by jessiagee** [comment]
This was integrated in AGX

**Pull request closed without merging**